src/content/reference/cluster-upgrades/_index.md

```diff
@@ -103,7 +103,7 @@ Your Kubernetes workloads should continue to work as expected (given they meet a
 
 In particular this means:
 
-- Nodes will be drained, then stopped, then recreated.
+- Nodes are drained and recreated at the same time.
 - As a consequence of draining, Pods running on a node will be rescheduled to other nodes.
 - Once the master node is taken down and recreated, the Kubernetes API will be unavailable for a short time.
```

Comment:
Based on nodes now being drained and recreated at the same time, does this still apply?

Reply:
This only applies if they are using single master

Reply:
can you suggest something Pau? I'm not really familiar with this area in the product :)

Reply:
sounds like i can leave it as is.